lib/ruby2d/window.rb

```diff
@@ -236,6 +223,36 @@ def add_object(o)
         false
       end
     end
-    
+
+    def events_store(event_type)
```

Comment:
What if we replaced all this with `instance_variable_get "@#{event_type}"`, maybe also renamed the method to `get_event_store` for clarity?